pkg/cmd/cli/cli.go

```diff
@@ -64,6 +64,7 @@ func NewCommandCLI(name, fullName string) *cobra.Command {
 
 	cmds.AddCommand(cmd.NewCmdLogin(f, in, out))
 	cmds.AddCommand(cmd.NewCmdProject(f, out))
+	cmds.AddCommand(cmd.NewCmdRequestProject("request-project", fullName+"request-project", f, out))
```

Comment:
Make the name "new-project" because users won't understand request.  You can leave the internal code named the same, just the end user facing name.